### PR TITLE
CAR2018-TN section needs discussion

### PR DESCRIPTION
**Pull request by robin-wang**
ITN section needs review and discussion.

### CODE REVIEW

File: 18_child_ITN.do [hunk 1]
```diff
@@ -7,6 +7,67 @@
 * 1) check the maximum value for tnln and use it as a maximum value for your "forvalues" of `t' and `j'
 * 2) check the tn15_* to find out the maximum amount of person recorded under each bednet and use it to determine the number of time you are going to replicate your "forvalues"
 
+	// CentralAfricanRepublic2018: tn5 - 11/12/13/14/15/16:other written down/36: other
+	// tnln - 1~10
+	// max value of tnln - 10; tn15_1-20; tn15_2-23; tn15_3-22; tn15_4-22
+if inlist(country_name,"CentralAfricanRepublic2018") {
```
Comment (robin-wang) [inline 18_child_ITN.do:13]: Section has not been updated from Guinea-Bissau2018 version, but top-banner description of key variables have been for CAR2018. The logic here is still very much murky, now that tn15_1-4 are slightly more inconsistent than before. 
@wengxyu1030

File: 1_antenatal_care.do [hunk 2]
```diff
@@ -157,21 +158,7 @@
 				replace c_anc_ski = . if `var' == "?"  // missing for DK/missing if no other skilled provider used
 			}	
 			replace c_anc_ski = . if bl2 != 1 | ~inrange(wb4,15,49)			// missing for births > 24 months ago
-		}
```
Comment (wengxyu1030) [inline 1_antenatal_care.do:160]: Could you confirm why moving sections for Tonga2019? Is this a rebase issue? @robin-wang

**Review by wengxyu1030 — COMMENTED** [review]
Thanks for your pull request, I have a small question on the code, appreciate your clarification.

File: 1_antenatal_care.do (hunk 2 above)
Comment (robin-wang) [reply 1_antenatal_care.do:160]: It's intended. 
Starting with line150, we have a section that's with ~Georgia, this is identical to the Tonga specific chunk. Hence the removal.

Comment (robin-wang) [reply 1_antenatal_care.do:160]: Similar treatment for bp\bs\ur in the anc section.

**Comment by robin-wang** [comment]
@wengxyu1030 Responded to question.
Thanks for debugging effort from you, inputs are now converted into output datafiles, saved in onedrive. Cheers!

**Comment by wengxyu1030** [comment]
Well noted, thanks for your response.

File: 18_child_ITN.do (hunk 1 above)
Comment (robin-wang) [reply 18_child_ITN.do:13]: @wengxyu1030 Just realised this one here...
ITN treatment for CAR2018 was quite elusive.. If you have a chance to look at it, may I get some thoughts from you about it?

**Pull request merged**